index.html

```diff
@@ -1,8 +1,13 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta http-equiv="Refresh" content="0; url='./speedrun-4.html'" />
+    <title></title>
   </head>
   <body>
+    <script>
+        //first check is for testing with localhost since there isn't a subdirectory.
+        if(window.location.href.lastIndexOf("/") == -1){window.location.replace(window.location.href + "speedrun-guide.html")}
```

Comment:
The meta tags were giving me troubles when redirecting since they would remove all the subdirectories for some reason so I opted to use js. Is there a performance issue with using js vs an html solution or would it be fine to leave it as is?

Reply:
just simpler. meta was working for me for both github and local.

Reply:
hmm well then there are two things that could be the problem on my end. Either chrome once again is being weird with things that work fine in firefox or maybe the redirect pages were using cached version that I could force refresh since it was instantly redirecting and I just didn't notice.